pkg/plugin/config.go

```diff
@@ -31,6 +31,12 @@ type PluginConfig struct {
 	GitHubAppInstallationID string
 	// The private Key PEM obtained for github app.
 	GitHubAppPrivateKeyPEM string
+
+	// GithubPlugInDisplayName is for display, e.g. for the web UI.
+	GithubPlugInDisplayName string
```

Comment:
Adding `GithubPlugin` prefix to avoid env var naming collision with other plugins.